mayastor/src/lvs/lvs_pool.rs

```diff
@@ -244,6 +257,85 @@ impl Lvs {
         }
     }
 
+    /// imports a pool based on its name, uuid and base bdev name
+    pub async fn import_from_args(args: PoolArgs) -> Result<Lvs, Error> {
+        if args.disks.len() != 1 {
```

Comment:
Within the try_from() parts, this is already checked, so this can not happen, can it?

Reply:
No, thinking a little ahead for lvm, the try_from only checks for 0 devices. Here we are checking devices should be exactly 1 in case of lvs.